frontend/src/Frontend/AppCfg.hs

```diff
@@ -6,6 +6,10 @@ import Language.Javascript.JSaddle (JSM)
 import Reflex.Dom
 import Kadena.SigningApi
 
+data EnabledSettings = EnabledSettings
+  { _enabledSettings_network :: Bool
```

Comment:
Not sure I understand why we have this type?

Reply:
The asana ticket mentions that we need some way of configuring turning settings on and off depending on what distro (app vs web) chainweaver is running in. The idea would be that they'd be a bool per setting and each app would turn it off or on. It's hard to really think about till we have the other settings, but it was moving in that direction. 

https://app.asana.com/0/944200340597880/1143451026068425/f

Reply:
Ahh I see. In the case of the network settings, we don't need this: it's enabled everywhere. The settings I was referring to there were the desktop specific bits. Can you remove this field and leave the settings as a stub? (Or remove them entirely, but I feel we will actually need something like this soon).